lib/action.rb

```diff
@@ -30,27 +30,19 @@ def check_version
   end
 
   def failed_status_description
-    text = "Update: #{file_path}"
+    text = failed_description || "Update: #{file_path}"
     text = text[0...137] + '...' unless text.length <= 140
     text
   end
 
-  def version_changed?
-    version_file_changed?(pull_number) && version_increased?(branch_name: head_branch, trunk_name: base_branch)
-  end
-
-  def version_file_changed?(pull_number)
-    file_changed = client.pull_request_files(repo, pull_number).map { |res| res[:filename] }
-    file_changed.include?(file_path)
-  end
-
   def version_increased?(branch_name:, trunk_name: 'master')
     branch_version = fetch_version_safe(ref: branch_name)
-    trunk_version = fetch_version(ref: trunk_name)
-    puts branch_version ? "branch version: #{branch_version}" : 'branch version: file not found, presumed name changed'
-    puts "trunk version: #{trunk_version}"
+    trunk_version = fetch_version_safe(ref: trunk_name)
+    return false if branch_version.nil? || trunk_version.nil?
 
-    branch_version.nil? || branch_version > trunk_version
+    puts "trunk version: #{trunk_version}"
+    puts "branch version: #{branch_version}"
```

Comment:
do we want to leave these puts statements in?

Reply:
That will help in debugging if github has something wiered going on